Adding Graphics

SarwaniM opened the pull request.

SarwaniM committed:
AddGraphics

ChandaSukesh commented on battleship.py:
```diff
@@ -34,7 +34,21 @@ def makeModel(data):
 '''
 def makeView(data, userCanvas, compCanvas):
-    return
+    def makeView(data, userCanvas, compCanvas):
```
No need of declaring two times.

SarwaniM replied:
done

SarwaniM committed:
changes done:
removed repated def of drawGrid()
changed drawGrid() in line 37 to test.testGrid()
line 116: changed 2 to ship_unclicked

ChandaSukesh commented on __pycache__/battleship.py:
```diff
@@ -0,0 +1,313 @@
+      for i in range(data["rows"]):
+        for j in range(data["cols"]):
+            if grid[i][j]==SHIP_UNCLICKED:
+               canvas.create_rectangle(j*50, i*50, (j+1)*50, (i+1)*50,fill="yellow")
```
Why are you hardcoding '50', make use of variables which you have declare above.

SarwaniM committed:
addShips,  makeMode,l makeView optimised
